install/tools/man/ipa-cacert-manage.1

```diff
@@ -31,7 +31,7 @@ ipa\-cacert\-manage \- Manage CA certificates in IPA
 \- Renew the IPA CA certificate
 .sp
 .RS
-This command can be used to manually renew the CA certificate of the IPA CA.
+This command can be used to manually renew the CA certificate of the IPA CA (caSigningCert cert-pki-ca). To renew other certificates, use getcert-resubmit(1).
```

Comment:
Please prefix the nickname with "NSS database nickname: " and double-quote the nickname itself. ACKing it when it's done.